src/main/java/net/apnic/whowas/rdap/GenericObject.java

```diff
@@ -97,6 +94,13 @@ public String getHandle()
         return getObjectKey().getObjectName();
     }
 
+    public List<Link> getLinks()
+    {
+        return Arrays.asList(
+            new RelativeLink("self", getObjectType().getPathSegment() + "/" + getPathHandle(),
+                RdapConstants.RDAP_MEDIA_TYPE.toString()));
```

Comment:
Is this not expected to change if a client is explicit with 'Accept: application/json' as opposed to 'Accept: application/rdap+json'?

Reply:
No reason to change from current implementation as per discussion.